docs/designers-developers/developers/themes/theme-json.md

```diff
@@ -348,6 +348,7 @@ These are the current color properties supported by blocks:
 | Post Title | Yes | Yes | - | Yes |
 | Site Tagline | Yes | Yes | - | Yes |
 | Site Title | Yes | Yes | - | Yes |
+| Social Links | Yes | - | - | Yes |
```

Comment:
:wave: While this PR added support for icon color & background, it doesn't use the block support mechanism, hence these properties can't be targeted via theme.json. I've prepared a fix for this at https://github.com/WordPress/gutenberg/pull/29294

Reply:
Thank you for the fix!